docs/tutorial/kubernetes-charm-go.rst

```diff
@@ -443,8 +443,15 @@ to send a request via the ingress. It should return the
     setting a DNS record.
 
 
-Configure the Go app
---------------------
+The development cycle
+---------------------
+
+So far, we have worked though the entire cycle, from creating an app to deploying it.
+But now – as in every real-world case – we will go through the experience of iterating to
+develop the app, and deploy each iteration.
```

Comment:
```suggestion
So far, we've worked though the entire cycle, from creating an app to deploying
it. But now -- as in every real-world case -- we'll go through the experience of
iteratively developing the app, and deploying each iteration.
```

The dashes are quite minor, but it's best if we let Sphinx handle them in the source.